persistence-cassandra-4.0/pom.xml

```diff
@@ -18,6 +18,20 @@
     -->
     <cassandra.bundled-driver.version>3.11.0</cassandra.bundled-driver.version>
   </properties>
+  <dependencyManagement>
+    <dependencies>
+      <!-- 29-Mar-2022, tatu: Need to force Netty overrides here
+        -->
+      <dependency>
+        <groupId>io.netty</groupId>
+        <artifactId>netty-bom</artifactId>
+        <version>${netty.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
```

Comment:
Will this work if there are no direct netty dependencies in the pom, meaning will it also affect the transitive deps?

Reply:
Yes.